### ISSUE TITLE
federation redirector: redirecting /versions path

### ISSUE THREAD

**Issue by bitnik** [jupyterhub/mybinder.org-deploy#989]
@betatim can we add `/versions` path to federation redirector? I am not sure if it makes sense, because I am not sure if both deployments always have the same versions.

I am asking this because today I updated our gallery (https://notebooks.gesis.org/gallery/), so it uses https://my.bndr.it to create badge/launch urls for mybinder. And it uses the same url to get versions info and https://my.bndr.it/versions returns 404, so we don't get versions info for mybinder.org:

![image](https://user-images.githubusercontent.com/6059799/58487434-dbc13e00-8167-11e9-8964-7d42c9208549.png)

Instead of adding redirection for `/versions`, maybe I should add 2 mybinders (https://mybinder.org and https://binder.mybinder.ovh) separately into binder drop-down list. What do you think?

**Comment by betatim** [comment]
I wouldn't use the my.bndr.it domain for anything but tests. The plan is to retire it once we have tested things a bit (that is why I used one of my private domains and not one that is part of mybinder.org :) ).

I am undecided if the "main federation domain" should redirect the `/versions` endpoint or not. Right now my feeling was that one shouldn't try and query the versions of the "main federation domain" but the individual hosts in it. I think the plan is to keep all hosts on the same version at all times but not sure if we are ready to make that promise yet. Thoughts welcome.

For now I'd add two hosts https://binder.mybinder.ovh/ and https://mybinder.org/ to the gallery. And then update one we have the federation actually up and running.
